haystack/nodes/answer_generator/openai.py

```diff
@@ -52,25 +53,29 @@ class OpenAIAnswerGenerator(BaseGenerator):
     def __init__(
         self,
         api_key: str,
-        model: str = "text-curie-001",
-        max_tokens: int = 13,
+        model: str = "text-davinci-003",
+        max_tokens: int = 50,
         top_k: int = 5,
         temperature: float = 0.2,
-        presence_penalty: float = -2.0,
-        frequency_penalty: float = -2.0,
+        presence_penalty: float = 0.1,
+        frequency_penalty: float = 0.1,
         examples_context: Optional[str] = None,
-        examples: Optional[List] = None,
-        stop_words: Optional[List] = None,
+        examples: Optional[List[List[str]]] = None,
+        stop_words: Optional[List[str]] = None,
         progress_bar: bool = True,
+        prompt_template: Optional[PromptTemplate] = None,
+        context_join_str: str = " ",
     ):
 
         """
         :param api_key: Your API key from OpenAI. It is required for this node to work.
         :param model: ID of the engine to use for generating the answer. You can select one of `"text-ada-001"`,
-                     `"text-babbage-001"`, `"text-curie-001"`, or `"text-davinci-002"`
+                     `"text-babbage-001"`, `"text-curie-001"`, or `"text-davinci-003"`
                      (from worst to best and from cheapest to most expensive). For more information about the models,
                      refer to the [OpenAI Documentation](https://beta.openai.com/docs/models/gpt-3).
-        :param max_tokens: The maximum number of tokens allowed for the generated Answer.
+        :param max_tokens: The maximum number of tokens reserved for the generated Answer. 
+                           A higher number allows for longer answers without exceeding the max prompt length of the OpenAI model.
+                           A lower number allows longer prompts with more documents passed as context, but the generated answer might be cut after max_tokens. 
```

Comment:
@sjrl update the docstring here as it wasn't obvious to users that max_tokens has this trade-off.